_posts/2021-10-19-Subnero-Vortice-partnership.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: post
+title: New business partner for Subnero in Republic of Korea
+banner : images/banner-pulse-standard.jpg
+date:  2021-07-08
```

Comment:
Is the date supposed to be backdated to 8th of July?

Reply:
Again, thank you for having an eye for details.